src/main/java/Krot.java

```diff
@@ -0,0 +1,146 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+
+public class Krot {
+    static String botName = "Krot"; // Static variable for the bot's name
+    static List<Task> tasks = new ArrayList<>(); // Static variable list of all the tasks
+    static boolean hasEnded = false; // Static variable to end the chatbot
+
+    public static String readInput() {
+        // Reads the input and returns the input as a string
```

Comment:
If this comment is about this function, consider using JavaDoc to comment about it. https://se-education.org/guides/conventions/java/basic.html#comments

Edit: same applies to the other functions you have commented on